### PR TITLE
XW-3140 | community page module on right rail

### PR DESCRIPTION
**Pull request by rybmat**
https://wikia-inc.atlassian.net/browse/XW-3140

@Wikia/x-wing

### CODE REVIEW

File: skins/oasis/modules/templates/CommunityPageEntryPointController_Index.php [hunk 1]
```diff
@@ -1,5 +1,15 @@
-<section class="community-page-entry-point-module module">
-	<div class="community-page-entry-point-logo"></div>
-	<div class="community-page-entry-point-description"><?= wfMessage( 'communitypage-help-us-grow' )->parse() ?></div>
-	<a href="<?= SpecialPage::getTitleFor( 'Community' )->getLocalURL(); ?>" class="community-page-entry-point-button"><?= wfMessage( 'communitypage-entry-button' )->escaped() ?></a>
+<section class="community-page-entry-point-module">
+	<div class="avatars">
+		<? foreach( $avatars as $index => $avatar ): ?>
+			<a href="<?= $avatar['userProfileUrl'] ?>">
+				<img class="wds-avatar" style="left:<?= ($index * -10) + ( count( $avatars ) - 1 ) * 5 ?>px; z-index: <?= 5 - $index ?>" src="<?= $avatar['avatarUrl'] ?>" />
```
Comment (latata) [inline skins/oasis/modules/templates/CommunityPageEntryPointController_Index.php]: maybe we could move these inline styles to scss or use desing system https://github.com/Wikia/design-system/blob/01699b6b06330d144c311ffd28cef332910a4ad5/style-guide/styles/wds-components/_avatar-stack.scss#L10-L10?

File: skins/oasis/js/Tracking.js [hunk 2]
```diff
@@ -748,7 +749,53 @@ jQuery(function ($) {
...
+
+			if (el.hasClass('more')) {
+				label = 'activity-more';
+			} else if (el.closest('.edited-by').length > 0) {
+				label = 'activity-username';
+			} else if (el.closest('em').length > 0) {
+				label = 'activity-title';
+			}
+
+			if (label !== undefined) {
```
Comment (latata) [inline skins/oasis/js/Tracking.js]: `if (label)` ?

File: skins/oasis/css/core/page.scss [hunk 3]
```diff
@@ -281,6 +279,60 @@
...
+
+	.activity-module-header {
+		border-bottom: 1px solid $color-page-border;
+		height: 30px;
+		margin-bottom: 8px;
+	}
+
+	.activity-icon {
+		margin-right: 6px;
+		fill: rgba($color-text, .75);
```
Comment (latata) [inline skins/oasis/css/core/page.scss]: alphabetical order ;)

**Review by latata — APPROVED** [review]

**Review by weronika — APPROVED** [review]

File: skins/oasis/css/core/page.scss [hunk 4]
```diff
@@ -281,6 +279,60 @@
 	}
 }
 
+.WikiaRail .activity-module {
+	background-color: transparent;
+	border: none;
```
Comment (latata) [inline skins/oasis/css/core/page.scss]: `border: 0;`

**Review by latata — APPROVED** [review]

File: extensions/wikia/Chat2/css/ChatWidget.scss [hunk 5]
```diff
@@ -84,104 +86,19 @@ body.skin-oasis .ChatMonobookWidget {
...
-		background: $color-module-background;
-		border: 1px solid $color-page-border;
-	}
-	margin: 0 auto;
-	margin-bottom: 10px;
-	padding: 16px 8px 8px;
+	margin: 0 auto 30px;
 	position: relative;
-	width: 284px;
+	width: 300px;
```
Comment (weronika) [inline extensions/wikia/Chat2/css/ChatWidget.scss:101]: I remember that we were to put this value in a separate variable while working on clean-up ticket :) it is used in 2 places

**Pull request merged**